packages/mural-canvas/src/lib/rpc.ts

```diff
@@ -181,6 +193,25 @@ export default class RpcClient extends EventEmitter {
     this.dispatch();
   };
 
+  rpc_callback(
```

Comment:
**naming**: I'm so sorry, IDK why I said to name this `rpc_callback`, I think we should follow camelCase and call this `rpcCallback`.

Let's fix that before people start using this interface.

It's probably my brain mapping the `mural.rpc_callback` message type.